utils/cli/src/table.rs

```diff
@@ -0,0 +1,189 @@
+use prettytable::{color, format, format::TableFormat, Attr, Cell, Row, Table};
+use std::collections::HashMap;
+use crate::cmd::CommandOutput;
+
+pub fn print_table(
+    columns: &Vec<String>,
+    values: &Vec<serde_json::Value>,
+    summary: &Vec<serde_json::Value>,
+) {
+    let mut table = Table::new();
+    //table.set_format(*format::consts::FORMAT_NO_LINESEP_WITH_TITLE);
```

Comment:
Removed it, but initially I left it there because I assumed it might have been left as some reminder to revert it back later or sth (this code was just moved and splitted).